ansible/playbooks/molecule/steamvr_docker/tests/test_default.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
+    f = host.file('/home/user/.steam/bin/vrstartup.sh')
+
+    assert f.exists
+    assert f.user == 'root'
```

Comment:
why file in the user's home directory is owned by root?

Reply:
Because steamvr is installed as root: https://github.com/shadow-robot/aurora/blob/ba0a916bfd7cb4b8b1de030194e0a1553dd6ba23/ansible/roles/installation/steamvr/tasks/main.yml#L81 
Last I checked it didn't work if the installer was ran as a normal user